### DISCUSSION TITLE
Notifications listener for activity/reputation points

### DISCUSSION THREAD

**Discussion by Ken-Andre** [dailydotdev/daily#1098] [Feature Requests]
When our posts, comments are upvoted or we gained points we saw a notification.
But not when it is the opposite or when we just loose these points without any reason. 
A notification when gaining or loosing points or anything will be great to resolve this.

**Comment by idoshamun** [comment]
Hi @Ken-Andre,

Thanks for the suggestion. We'll consider how to proceed 🙏
